core/examples/luigi-example-svelte/package.json

```
@@ -10,27 +10,28 @@
     "node-sass": "^4.13.1",
     "serve": "^11.0.0",
     "style-loader": "^0.23.1",
-    "svelte": "^3.0.0",
+    "svelte": "^3.20.1",
     "svelte-loader": "2.13.3",
-    "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.0",
-    "webpack-dev-server": "^3.3.1"
+    "webpack": "^4.42.1",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.10.3"
   },
   "scripts": {
-    "build:prod": "webpack --mode=production --display-error-details",
+    "build:prod": "NODE_ENV=production webpack --display-error-details",
     "build:dev": "webpack",
-    "dev": "concurrently \"npm run build:dev -- --watch\" \"npm run serve:dev\" \"npm run serve:luigi\"",
-    "serve": "concurrently \"npm run serve:dev\" \"npm run serve:luigi\"",
-    "serve:dev": "sirv public --single --dev --port 5000",
-    "serve:luigi": "cd public && ws --spa mf.html --port 5001"
+    "dev": "concurrently \"npm run build:dev --watch\" \"npm run serve:luigi\" \"npm run serve:microfrontend\"",
```

> [0] npm run build:dev --watch exited with code 0

--watch is not watching anymore, is there a reason for the change?
```suggestion
    "dev": "concurrently \"npm run build:dev -- --watch\" \"npm run serve:luigi\" \"npm run serve:microfrontend\"",
```

---

Actually an idea was to remove `--watch` as there is no sense to watch file as we have no `hot reload` option running and in the case of changes we have to run `npm run dev` again.

Also, after changes when `--watch` is enabled, it was resulting in my case with 404 error when tried to reload page after changes. Sometimes I had to run `npm run dev` 2 times.

---

got the same bahaviour

---

I also have seen this, caused by Cleanup webpack plugin. I have removed it since its not ultimately required for the example app.